service/nodeconfig/v1/resource/node/create.go

```diff
@@ -47,13 +52,35 @@ func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createChange inte
 		return microerror.Mask(err)
 	}
 
-	nodes, err := k8sClient.CoreV1().Nodes().List(apismetav1.ListOptions{})
-	if err != nil {
-		return microerror.Mask(err)
-	}
+	{
+		n := key.NodeName(customObject)
+		t := types.StrategicMergePatchType
+		p := []byte(UnschedulablePatch)
 
-	for _, n := range nodes.Items {
-		fmt.Printf("%#v\n", n)
+		{
+			manifest, err := k8sClient.CoreV1().Nodes().Get(n, apismetav1.GetOptions{})
+			if err != nil {
+				return microerror.Mask(err)
+			}
+			fmt.Printf("unschedulable\n")
+			fmt.Printf("%#v\n", manifest.Spec.Unschedulable)
+			fmt.Printf("unschedulable\n")
+		}
```

Comment:
This is all for debugging. I would like to keep this now since I want to restructure the logic at the end anyway. I am now more trying to figure out how to do things. I try to make it beautiful later.